README.md

```diff
@@ -1,24 +1,61 @@
-# README
+# ExpenseTracker
+> This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Screenshot
+![screenshot](screenshot.png)
 
-Things you may want to cover:
+[Live demo](https://infinite-tor-95974.herokuapp.com/)
+# Built With
 
-* Ruby version
+- Ruby
+- Rspec
+- Rails
+- PostgreSQL
 
-* System dependencies
+## Getting Started
 
-* Configuration
+To get a local copy up and running follow these simple example steps.
 
-* Database creation
+### Prerequisites
 
-* Database initialization
+- [Ruby](https://www.ruby-lang.org/)
+- [Ruby on Rails](https://rubyonrails.org/)
 
-* How to run the test suite
+### Setup
+- Clone or download this repo on your machine
+- Enter project directory
+- Run `install bundle` to install dependencies
+- Run  `rake db:create` to create a database
+- Run  `rake db:migrate` to create a migrations
```

Comment:
- [ ] Great readme. But, I think you should add `npm install` here, because someone who clones your repo would get `webpack` error if he/she doesn't run run `npm install`. :pray: